src/Pim/Component/Catalog/Builder/ProductBuilder.php

```diff
@@ -93,6 +93,8 @@ public function createProduct($identifier = null, $familyCode = null)
 
         if (null !== $identifier) {
             $product->setIdentifier($identifier);
+            $identifierAttribute = $this->attributeRepository->getIdentifier();
```

Comment:
we can't plug the `LoadProductValuesListener` on the `ProductEvents::CREATE` event?